index.html

```diff
@@ -2567,7 +2567,7 @@ <h2>Definition of Roles</h2>
 			<rdef>definition</rdef>
 			<div class="role-description">
 				<p>A definition of a term or concept. See related <rref>term</rref>.</p>
-				<p>Authors SHOULD identify the <a>element</a> being defined by giving that element a role of <rref>term</rref> and referencing it with the <pref>aria-labelledby</pref> <a>attribute</a> or by making the element with role <rref>term</rref> a descendant of the element with role <code>definition</code>.</p>
+				<p>Authors SHOULD identify the <a>element</a> being defined by giving that element a role of <rref>term</rref> and referencing it with the <pref>aria-details</pref> <a>attribute</a> or by making the element with role <rref>term</rref> a descendant of the element with role <code>definition</code>.</p>
```

Comment:
Seems ancillary to this pull request. @joanmarie  would this need any corresponding CORE-AAM or implementation changes? If so I suggest moving this to a separate PR.

Reply:
Doesn't require implementation changes in browser, or any CORE-AAM changes that I would be aware of.

I can put in separate change, but it did seem related to having a more consistent understanding of what aria-details does.

Reply:
I agree that there is no need to change Core-AAM or user agent implementations.

But I tend to agree with James that this change does seem related, but not part of the aria-details revamp.

Reply:
Ok I will separate.

Reply:
Created https://github.com/w3c/aria/pull/1147